src/main/java/edu/kis/powp/jobs2d/command/io/ICommandLoader.java

```diff
@@ -0,0 +1,9 @@
+package edu.kis.powp.jobs2d.command.io;
+
+import edu.kis.powp.jobs2d.command.DriverCommand;
+
+import java.util.List;
+
+public interface ICommandLoader {
+	List<DriverCommand> loadCommandsFromFile(String fileData);
```

Comment:
importing should be done from a text, not from a file. Reading text from a file is a separate task (responsibility)